Can't install

Hi, your lib seems to be what I need, but I can't install:

With brew:

```
==> Building SourceDocs
==> swift build --disable-sandbox -c release -Xswiftc -static-stdlib


Last 15 lines from /Users/ak/Library/Logs/Homebrew/sourcedocs/01.swift:
Resolving https://github.com/norio-nomura/SourceKit.git at 1.0.1
Cloning https://github.com/antitypical/Result.git
Resolving https://github.com/antitypical/Result.git at 3.2.4
Cloning https://github.com/carthage/Commandant.git
Resolving https://github.com/carthage/Commandant.git at 0.12.0
Cloning https://github.com/onevcat/Rainbow
Resolving https://github.com/onevcat/Rainbow at 3.0.0
Cloning https://github.com/eneko/MarkdownGenerator.git
Resolving https://github.com/eneko/MarkdownGenerator.git at 0.2.1
Cloning https://github.com/drmohundro/SWXMLHash.git
Resolving https://github.com/drmohundro/SWXMLHash.git at 4.2.3
Cloning https://github.com/norio-nomura/Clang_C.git
Resolving https://github.com/norio-nomura/Clang_C.git at 1.0.2
Cloning https://github.com/thoughtbot/Curry.git
Resolving https://github.com/thoughtbot/Curry.git at 3.0.0

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/eneko/homebrew-tap/issues
```

With make:
```
make
swift build --disable-sandbox -c release -Xswiftc -static-stdlib
Fetching https://github.com/jpsim/SourceKitten.git
Fetching https://github.com/carthage/Commandant.git
Fetching https://github.com/eneko/MarkdownGenerator.git
Fetching https://github.com/onevcat/Rainbow
Fetching https://github.com/thoughtbot/Curry.git
Fetching https://github.com/jpsim/Yams.git
Fetching https://github.com/drmohundro/SWXMLHash.git
Fetching https://github.com/norio-nomura/Clang_C.git
Fetching https://github.com/Carthage/Commandant.git
Fetching https://github.com/antitypical/Result.git
Fetching https://github.com/norio-nomura/SourceKit.git
Cloning https://github.com/Carthage/Commandant.git
Resolving https://github.com/Carthage/Commandant.git at 0.12.0
Cloning https://github.com/jpsim/SourceKitten.git
Resolving https://github.com/jpsim/SourceKitten.git at 0.18.2
Cloning https://github.com/jpsim/Yams.git
Resolving https://github.com/jpsim/Yams.git at 0.3.7
Cloning https://github.com/norio-nomura/SourceKit.git
Resolving https://github.com/norio-nomura/SourceKit.git at 1.0.1
Cloning https://github.com/antitypical/Result.git
Resolving https://github.com/antitypical/Result.git at 3.2.4
Cloning https://github.com/carthage/Commandant.git
Resolving https://github.com/carthage/Commandant.git at 0.12.0
Cloning https://github.com/onevcat/Rainbow
Resolving https://github.com/onevcat/Rainbow at 3.0.0
Cloning https://github.com/eneko/MarkdownGenerator.git
Resolving https://github.com/eneko/MarkdownGenerator.git at 0.3.0
Cloning https://github.com/drmohundro/SWXMLHash.git
Resolving https://github.com/drmohundro/SWXMLHash.git at 4.2.3
Cloning https://github.com/norio-nomura/Clang_C.git
Resolving https://github.com/norio-nomura/Clang_C.git at 1.0.2
Cloning https://github.com/thoughtbot/Curry.git
Resolving https://github.com/thoughtbot/Curry.git at 3.0.0
error: Found multiple packages with the name Commandant: https://github.com/carthage/Commandant.git, https://github.com/Carthage/Commandant.git
make: *** [build] Error 1

```

--- a/Package.swift
+++ b/Package.swift
@@ -8,7 +8,7 @@ let package = Package(
     ],
     dependencies: [
         .package(url: "https://github.com/jpsim/SourceKitten.git", from: "0.18.0"),
-        .package(url: "https://github.com/carthage/Commandant.git", from: "0.12.0"),
+        .package(url: "https://github.com/Carthage/Commandant.git", from: "0.12.0"),
         .package(url: "https://github.com/eneko/MarkdownGenerator.git", from: "0.1.0"),
         .package(url: "https://github.com/onevcat/Rainbow", from: "3.0.0"),
         .package(url: "https://github.com/thoughtbot/Curry.git", from: "3.0.0"),

